src/math-translator.js

```diff
@@ -229,6 +255,14 @@ function normalizeTranslatedMath(math, lang) {
          // We cannot allow a literal space here, cause Tex would ignore it
          {langs: MATH_RULES_LOCALES.THOUSAND_SEP_AS_THIN_SPACE,
             regex: /([0-9])\{?~\}?([0-9])(?=[0-9]{2})/g, replace: '$1\\,$2'},
+
+         // KaTeX seem to support a bit more trig functions then default LaTeX
+         // but we're using operatornames for them as well
+         // https://katex.org/?data=%7B%22displayMode%22%3Atrue%2C%22leqno%22%3Afalse%2C%22fleqn%22%3Afalse%2C%22throwOnError%22%3Atrue%2C%22errorColor%22%3A%22%23cc0000%22%2C%22strict%22%3A%22warn%22%2C%22trust%22%3Afalse%2C%22macros%22%3A%7B%22%5C%5Cf%22%3A%22f(%231)%22%7D%2C%22code%22%3A%22%25%20%5C%5Cf%20is%20defined%20as%20f(%231)%20using%20the%20macro%5Cn%5C%5Csin%20%5C%5Carcsin%20%5C%5Ccos%20%5C%5Carccos%20%5C%5C%5C%5C%20%5C%5Ctan%20%5C%5Carctan%20%5C%5Ctg%20%5C%5Carctg%20%5C%5C%5C%5C%5Cn%5C%5Ccot%20%5C%5Ccotg%20%5C%5Cctg%20%5C%5Coperatorname%7Barccot%7D%20%5C%5C%5C%5C%5Cn%5C%5Ccsc%20%5C%5Ccosec%20%5C%5Coperatorname%7Barccsc%7D%20%5C%5C%5C%5C%5Cn%5C%5Coperatorname%7Bsen%7D%7Bx%7D%20%5C%5Csin%7Bx%7D%20%5C%5Csin%20x%22%7D
+         // Do it for all langs, probably no need to be specific here
+         {langs: [].concat(...Object.values(MATH_RULES_LOCALES)),
```

Comment:
Can you explain why this rule can be applied to all locales?

This is also repeating the same locales multiple times. You can change it to `Array.from(new Set([].concat(...Object.values(MATH_RULES_LOCALES))))` so that a locale will appear only once in the final list.

Reply:
We're inside normalizeTranslatedMath, which ensures that if translator used a slightly different but essentially equivalent syntax, ST will still work for them. It's okay to apply to all because ST will fail anyway if translator translated trig function that we did not expect. We could be specific here and selectively concat only lists from MATH_RULES_LOCALES that pertain to trig functions. This was easier (and future proof in case we ever added more rules for trig functions), but LMK if you think I should change it.

I did think about the duplicates in the list, but since this is meant to be local and is used only inside `.includes(lang)` if I thought it's okay. Your call.

Reply:
@mpolyak I changed this and explicitly concat required rules. There's a precedence to this in `translateNumbers()` for thousand separators.

https://github.com/Khan/translation-assistant/blob/244ad4412aad586ee59ed18e08ba33c54e519672/src/math-translator.js#L76

Reply:
@mpolyak okay, done.